services/ui-src/src/components/pages/Export/McparPdfExport.tsx

```diff
@@ -0,0 +1,126 @@
+import { Box, Heading } from "@chakra-ui/react";
+import { StickyBanner, ReportContext, Table, FieldsSection } from "components";
+
+import { useContext } from "react";
+import { convertDateUtcToEt } from "utils";
+
+import addEditProgram from "../../../forms/addEditProgram/addEditProgram.json";
```

Comment:
Suggestion: For this text and all other text on this page, we should make a separate .json file with text for it so we can provide that json file to the component and that way we can reuse this for MLR and NAAAR in the future as needed.